packages/storybook/stories/Adapters/snippets/ChatAdapterExample.snippet.tsx

```diff
@@ -11,32 +11,17 @@ type ChatAdapterExampleProps = {
 };
 
 export const ChatAdapterExample = (props: ChatAdapterExampleProps): JSX.Element => {
-  const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
-  useEffect(() => {
-    if (props) {
-      const createAdapter = async (): Promise<void> => {
-        setChatAdapter(
-          await createAzureCommunicationChatAdapter({
-            endpoint: props.endpointUrl,
-            userId: props.userId,
-            displayName: props.displayName,
-            credential: new AzureCommunicationTokenCredential(props.accessToken),
-            threadId: props.threadId
-          })
-        );
-      };
-      createAdapter();
-    }
-    return () => {
-      if (chatAdapter) {
-        chatAdapter.dispose();
-      }
-    };
-  }, [props, chatAdapter]);
-
+  const credential = useMemo(() => new AzureCommunicationTokenCredential(props.accessToken), [props.accessToken]);
+  const adapter = useAzureCommunicationChatAdapter({
+    endpoint: props.endpointUrl,
```

Comment:
i dont think this is something we need to solve in this PR but have we consolidated on endpoint or endpointUrl? or do we need to keep them the same?

Reply:
I think `endpoint` is better, but not touching that here.